src/form/Form.tsx

```diff
@@ -587,28 +608,30 @@ function Form({
 
     initState.validateCallbacks[formKey] = (trigger: any) => {
       const inlineErrors = {};
-      const errors = newStep.servar_fields
+      const errors = [...newStep.servar_fields, ...newStep.buttons]
```

Comment:
could this instead call validateElements directly? stuff like filtering out hidden elements is already handled by that function.

Reply:
Refactored validateElements and used it here as well.